### DISCUSSION TITLE
python-mint available on PyPI

### DISCUSSION THREAD

**Discussion by bjlittle** [pletzer/mint#31] [General]
@pletzer I grabbed the `python-mint` namespace on PyPI, just incase.

Totally happy to close this PyPI project so that you can grab it an own it; it is your project after all :wink: 

See https://pypi.org/project/python-mint/

**Comment by bjlittle** [comment]
I only pushed the `sdist` of `mint` 1.8.0 to `python-mint` on PyPI.

I'm not able to build a [PEP513](https://www.python.org/dev/peps/pep-0513/) compatible wheel for `mint` as my toolchain is too modern and it pulls in additional libraries.

We could pursue attempting to make the `bdist_wheel` of `mint` compliant, but that will require some extra effort.

At the moment I can install the `sdist` of `mint` from PyPI reasonably easily, which seems sufficient enough. What do you think?

Reference: https://github.com/pypa/manylinux

**Reply by bjlittle** [reply]
@pletzer Can you confirm that `pip install python-mint` is working for you on `osx`?

Thanks 👍

**Reply by pletzer** [reply]
@bjlittle Works on OS X with python 3.8! Thanks much!
